### ISSUE TITLE
RSS-Feed Implementation

### ISSUE THREAD

**Issue by Lukas2112** [eknoes/covidbot#121]
There may be people who do not use any of the currently supported messengers and may rely on RSS as a source of information. So if it were technically possible, it could also make sense to implement it.

**Comment by eknoes** [comment]
Like one personal RSS Feed per User?

**Comment by Lukas2112** [comment]
Perhaps it would be possible to create an RSS feed script for each postcode and each federal state using a Python script? Unfortunately, I am not (yet) so advanced in Python programming that I can reliably assess this.

**Comment by eknoes** [comment]
Yes, would be possible. Won't be our top priority, but we already think about implementing some web-based services, so if we do that anyway we could easily integrate RSS Feeds.

#99

**Comment by Lukas2112** [comment]
I remain curious! :)
